API/posts/PostController.js

```diff
@@ -135,7 +135,7 @@ exports.view = function (req, res, next) {
         }
         console.log('Post OP: ' + temp.userID);
         try {
-            userObjectId = mongoose.Types.ObjectId(temp.userID);
+            userObjectId = mongoose.Types.ObjectId("5e75532986559b1e8cba2959");
```

Comment:
Is the user id meant to be hardcoded here?

Reply:
no, this is wrong